samples/RayTracing/CameraManipulator.hpp

```diff
@@ -29,7 +29,7 @@ namespace vk
         enum class Mode { Examine, Fly, Walk, Trackball };
         enum class MouseButton { None, Left, Middle, Right };
         enum class ModifierFlagBits { Shift = 1, Ctrl = 2, Alt = 4 };
-        using ModifierFlags = vk::Flags<ModifierFlagBits, uint32_t, ModifierFlagBits::Shift>;
+        using ModifierFlags = vk::Flags<ModifierFlagBits, uint32_t>;
```

Comment:
vk::Flags need the default initialization value as the third template parameter now.

Reply:
But doesn't that break compilation for old Vulkan SDK users?
I'd like to ask reasons for breaking change, rather than doing like this:
```cpp
  template <typename BitType, typename MaskType = VkFlags, BitType defaultValue = BitType()>
  class Flags
```

Reply:
That will break, but only for those who introduce their own Flags.
But you're right, and it's easy to prevent the breaking, so I'll add the default for the defaultValue.